### ISSUE TITLE
resolve http post issue by getting Beth to purchase SSL cert

### ISSUE THREAD

**Issue by vomc** [vomc/csr-mapping-engine#26]

**Comment by vomc** [comment]
cert was purchased this weekend, but can't find the option to configure it... asked blueHost today

**Comment by vomc** [comment]
checked with BlueHost support today, no answer yet.

**Comment by vomc** [comment]
still waiting on final confirm from dreamhost, followed up with another ticket today. Tool is working at https://amman.clustermappinginitiative.org/geotool/geo3.html#
